src/lib/elements/forms/inputSearch.svelte

```diff
@@ -26,13 +27,19 @@
         }
     });
 
-    const valueChange = (event: Event) => {
+    function valueChange(event: Event) {
         clearTimeout(timer);
         timer = setTimeout(() => {
             const target = event.target as HTMLInputElement;
             value = target.value;
+            dispatch('change', value);
         }, debounce);
-    };
+    }
+
+    function clearSearch() {
+        value = '';
+        dispatch('clear');
```

Comment:
do we use constants for dispatched event names in other parts of the code? If yes we should keep it consistent

Reply:
Usually we handle it like this :+1: